cli/cmd/erc721.js

```diff
@@ -53,48 +68,38 @@ const transferCmd = new Command("transfer")
         setupParentArgs(args, args.parent.parent)
 
         // Instances
-        const erc721Instance = new ethers.Contract(args.erc721Address, ERC721Contract.abi, args.wallet);
-        const bridgeInstance = new ethers.Contract(args.bridgeAddress, BridgeContract.abi, args.wallet);
-        const erc721HandlerInstance = new ethers.Contract(args.erc721HandlerAddress, ERC721HandlerContract.abi, args.wallet);
+        const erc721Instance = new ethers.Contract(args.erc721Address, constants.ContractABIs.Erc721Mintable.abi, args.wallet);
+        const bridgeInstance = new ethers.Contract(args.bridgeAddress, constants.ContractABIs.Bridge.abi, args.wallet);
+        const erc721HandlerInstance = new ethers.Contract(args.erc721HandlerAddress, constants.ContractABIs.Erc721Handler.abi, args.wallet);
 
         // Approve tokens
         await erc721Instance.approve(args.erc721HandlerAddress, args.id);
-        console.log("[ERC721 Transfer] Approved tokens!");
         console.log(`[ERC721 Transfer] Approved ${args.erc721HandlerAddress} to move ${args.id} on behalf of ${args.wallet.address}!`);
 
-        // Check pre balance
-        const depositerPreBal = await erc721Instance.balanceOf(args.wallet.address);
-        const handlerPreBal = await erc721Instance.balanceOf(args.erc721HandlerAddress);
-        console.log(`[ERC721 Transfer] Depositer ${args.wallet.address} owns ${depositerPreBal.toNumber()} tokens `);
-        console.log(`[ERC721 Transfer] Handler ${args.erc721HandlerAddress} owns ${handlerPreBal.toNumber()}`);
-
-
         // Compute resourceID
-        resourceID = await erc721HandlerInstance._tokenContractAddressToResourceID(args.erc721Address)
+        const resourceID = await erc721HandlerInstance._tokenContractAddressToResourceID(args.erc721Address)
 
-        const data = '0x' +
-            resourceID.substr(2) +              // OriginHandlerAddress  (32 bytes)          
-            ethers.utils.hexZeroPad(ethers.utils.hexlify(args.id), 32).substr(2) +      // Token ID
-            ethers.utils.hexZeroPad(ethers.utils.hexlify(32), 32).substr(2) +    // len(recipientAddress) (32 bytes)
-            ethers.utils.hexZeroPad(args.recipient, 32).substr(2);                    // recipientAddress      (?? bytes)
+        const depositData = '0x' +
+            resourceID.substr(2) +                                                  // resourceID            (32 bytes) for now
+            ethers.utils.hexZeroPad(ethers.utils.hexlify(args.id), 32).substr(2) +  // Deposit Amount        (32 bytes)
+            ethers.utils.hexZeroPad(ethers.utils.hexlify(20), 32).substr(2) +       // len(recipientAddress) (32 bytes)
+            ethers.utils.hexlify(args.recipient).substr(2) +                // recipientAddress      (?? bytes)
```

Comment:
32 bytes no?

Reply:
recipient is 20 bytes, substr to remove `0x`